tests/ui/arch/read_clock_khr.rs

```diff
@@ -0,0 +1,16 @@
+// build-pass
+// compile-flags: -Ctarget-feature=+Int64,+ShaderClockKHR,+ext:SPV_KHR_shader_clock
```

Comment:
Ah, shoot, right, I'm silly and completely missed this the first time around, there should also be a `#[cfg(and(target_feature = "ShaderClockKHR", target_feature = "ext:SPV_KHR_shader_clock"))]` put on too, to both functions, sorry!